src/signals/incident-management/containers/StandardTextsAdmin/components/OverviewPage/OverviewPage.tsx

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { useEffect, useState } from 'react'
+
+import { Column, Row } from '@amsterdam/asc-ui'
+import { useDispatch } from 'react-redux'
+
+import LoadingIndicator from 'components/LoadingIndicator'
+import { showGlobalNotification } from 'containers/App/actions'
+import { VARIANT_ERROR, TYPE_LOCAL } from 'containers/Notification/constants'
+import { useFetch } from 'hooks'
+import { getErrorMessage } from 'shared/services/api/api'
+import configuration from 'shared/services/configuration/configuration'
+
+import { StyledColumn, StyledPagination, StyledButton } from './styled'
+import { useStandardTextAdminContext } from '../../context'
+import type { StandardText, StandardTextsData } from '../../types'
+import { Summary } from '../Summary'
+
+const PAGE_SIZE = 15
+
+export const OverviewPage = () => {
+  const dispatch = useDispatch()
+  const [standardTexts, setStandardTexts] = useState<StandardText[]>()
+
+  const { page, setPage } = useStandardTextAdminContext()
+
+  const { get, data, isLoading, error } = useFetch<StandardTextsData>()
+
+  useEffect(() => {
+    if (!standardTexts) {
+      get(configuration.STANDARD_TEXTS_SEARCH_ENDPOINT)
+    }
+  }, [get, standardTexts])
+
+  useEffect(() => {
+    data && setStandardTexts(data.results)
```

Comment:
why add and extra state here, and not just render data.results?

Reply:
it adds an extra render, not really a perf hit, just don't see a reason

Reply:
true true.